CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 IMPROVEMENTS:
 
 * auth/jwt: Add support for fetching groups and user information from G Suite during authentication. [[GH-123](https://github.com/hashicorp/vault-plugin-auth-jwt/pull/123)]
+* secrets/openldap: Add "ad" schema that allows the engine to correctly rotate AD passwords.
```

Comment:
Please link to the relevant PR.

Reply:
Oops I didn't look properly at the other links and thought they were links to GitHub issues. Thanks, done.